content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+
+![Prometheus Overview](/img/prometheus-1.0.0-beta-revwip.png)
+
+In this tutorial we'll learn how to deploy Prometheus, Grafana and the HPE Storage Array Exporter for Prometheus using nothing but Docker. A supported storage backend, such as a HPE Alletra, Nimble Storage, Primera or 3PAR is needed to gather metrics for visualization. These prerequisites are assumed along with basic docker and container knowledge. While this is a highly opinionated tutorial, the different components may be run standalone on Windows or Linux and may also be deployed directly on Kubernetes with Helm.
+
+# Get started
+
+The tutorial assumes docker is installed and functional. Credentials are also needed for a backend system. HPE Nimble Storage is being used in the examples and metrics may be named differently when using something else. Still, principles remain the same.
+
+To allow the containers to communicate with each other by name, a separate bridge network is created.
+
+```markdown
+docker network create prometheus
+```
+
+Note that all containers created below do not utilize any persistent storage at all. Any dashboards that have been created or metrics that have been gathered will not be accessible after a container restart.
+
+**Note:** This tutorial is purely for educational purposes and not intended for production deployments.
+
+# Run HPE Storage Array Exporter for Prometheus
+
+Let's begin by starting the HPE Storage Array Exporter for Prometheus container. It's a passive background process that responds to HTTP requests made by Prometheus. It also requires a backend configuration file. In the case of having multiple backends, one container and configuration file is needed for each one.
+
+Create a file named `my-array-1.yaml`:
+
+```yaml
+address: my-array-1
+username: admin
+password: admin
+```
+
+The `address` value can be an IP address or hostname resolvable in DNS. Then, run `docker`:
+
+```markdown
+docker run -d --rm --name my-array-1 \
+     -v $(pwd)/my-array-1.yaml:/etc/config/storage-system.yaml \
+     --network prometheus \
+     quay.io/hpestorage/array-exporter:v1.0.0-beta \
+     --log.path /var/log/hpe-array-exporter.log \
+     --accept-eula \
+     /etc/config/storage-system.yaml
+```
+
+By running the above command with the `--accept-eula` parameter, the end-user accepts the HPE End User License Agreement at <https://www.hpe.com/us/en/software/licensing.html>.
+
+The exporter is now up and running listening on port 8080. To ensure we can reach the metrics endpoint, run the following `docker` command:
+
+```markdown
+docker run --rm --network prometheus \
+    curlimages/curl -s http://my-array-1:8080/metrics
+```
+
+It should produce an output that enumerates the metric keys, value, type and brief help. Example:
```

Comment:
It may just be me, but "brief help" feels as if it's missing something. 
Also, should "Example:" be "For example:"?

Reply:
"HELP" indicates the metric help.
"TYPE" indicates what kind of metric type it is. "gauge" in this case.
Then we have the metric key and value.

Change "Example:" to "An example:".